web/src/beta/features/ProjectSettings/innerPages/PublicSettings/AliasSettings/index.tsx

```diff
@@ -124,11 +124,11 @@ const AliasSetting: FC<AliasSettingProps> = ({
         </UrlWrapper>
         <Button
           appearance="secondary"
-          title={t("clean")}
+          title={t("Reset")}
           icon="pencilLine"
           size="small"
           disabled={isDisabled}
-          iconColor={theme.content.weak}
+          iconColor={isDisabled ? theme.content.weak : theme.content.main}
```

Comment:
Basically i think disabled should effect the icon's color internally since we already manage the disabled appreance on Button, could you please give the color to icon base on disabled on `Button` instead of give it here?